lib/game.rb

```diff
@@ -0,0 +1,120 @@
+require 'pry'
+
+class Game
+  attr_accessor :board, :player_1, :player_2
+
+  # Defining WIN_COMBINATIONS constant
+  WIN_COMBINATIONS = [
+    [0,1,2], # top row
+    [3,4,5], # middle row
+    [6,7,8], # bottom row
+    [0,3,6], # left column
+    [1,4,7], # middle column
+    [2,5,8], # right column
+    [0,4,8], # diagonal 1
+    [2,4,6] # diagonal 2
+  ]
+
+  #Initialized a new game with two players and a new board
+  def initialize(player_1=Players::Human.new("X"), player_2=Players::Human.new("O"), board=Board.new)
+    @player_1 = player_1
+    @player_2 = player_2
+    @board = board
+  end
+
+  #Determines the current character by judging the number of plays made
+  def current_player
```

Comment:
we could define this simply with a ternary operator `?` and remove setting the variable

`@board.turn_count.even? ? @player_1 : @player_2`